endpoints/wishes/recieve.go

```diff
@@ -0,0 +1,22 @@
+package wishes
+
+import (
+	"net/http"
+
+	"github.com/bredbrains/tthk-wish-list/database"
+	"github.com/bredbrains/tthk-wish-list/models"
+	"github.com/gin-gonic/gin"
+)
+
+func Receive(c *gin.Context) {
+	var wishes []models.Wish
```

Comment:
Where we can to get user ID?
Authorization?
Request token to recieve wishes by user for security.